### PR TITLE
Allow changelogger regex to find directories with woocommerce* names 

### PR DESCRIPTION
**Pull request by opr**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

Changes the regex used to find package names in the `changelogger` tool.

Previously the command to generate changelogs would break if the repository was not in a directory called `woocommerce`. Since I have my repo in a folder called `woocommerce-monorepo` this caused an error when running `./tools/package-release/bin/dev prepare @woocommerce/extend-cart-checkout-block`

- [ ] This PR is a very minor change/addition and does not require testing instructions (if checked you can ignore/remove the next section).

### How to test the changes in this Pull Request:

1. Clone this repository into a directory called `woocommerce-monorepo`.
2. Add a change file to the `extend-cart-checkout-block` package. Example formatting and location: https://github.com/woocommerce/woocommerce/blob/c539e5614fab45358b6641af2e6b4811b61205e4/packages/js/extend-cart-checkout-block/changelog/add-inner-block-example 
2. Run `./tools/package-release/bin/dev prepare @woocommerce/extend-cart-checkout-block` from the repo root.
3. Ensure the changelog file generates correctly and no errors occur.

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [ ] Have you created a changelog file for each project being changed, ie `pnpm --filter=<project> changelog add`?
- **I couldn't see anywhere to add changelogs for this tool**

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: tools/changelogger/class-package-formatter.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ public function getEpilogue() {
 	 */
 	public function getReleaseLink( $version ) {
 		// Catpure anything past /woocommerce in the current working directory.
-		preg_match( '/\/woocommerce\/packages\/js\/(.+)/', getcwd(), $path );
+		preg_match( '/\/woocommerce[^\/]*\/packages\/js\/(.+)/', getcwd(), $path );
```
Comment (psealock) [inline tools/changelogger/class-package-formatter.php]: I see how this is an improvement, but it raises the notion that someone could name the Monorepo whatever they'd like. Should we take it one step further and make the regex `/\/packages\/js\/(.+)/`?

Comment (opr) [reply tools/changelogger/class-package-formatter.php]: Good point, I think it's fine to allow that, I've updated the regex. Thanks

**Review by psealock — DISMISSED** [review]
Thanks @opr, this is testing well

**Comment by opr** [comment]
@psealock I rebased this and would like to merge it, I forgot that it was still pending review. If you would approve it that would be great! thank you.

**Review by psealock — APPROVED** [review]
👍🏽

**Pull request merged**